File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyListViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewModelMyListViewElementsRow.cs
/// </filename>


namespace commands
{
    public class SelectRowCommand_ViewModelMyListViewElementsRow
  {
        private string HeaderLabelText;
     
        public     virtual string getHeaderLabelText()
    {
      return this.HeaderLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyListViewElementsRow.cpp
#include "SelectRowCommand_ViewModelMyListViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewModelMyListViewElementsRow::getHeaderLabelText()
  {
    return this->HeaderLabelText;
  }
  int SelectRowCommand_ViewModelMyListViewElementsRow::getRowIndex()
  {
    return this->RowIndex;
  }
}
